install/package_odoo_9.0_10.0.sh

```diff
@@ -2,6 +2,8 @@
 set -eo pipefail
 
 apt-get update
+# authentication error for libc-ares2
```

Comment:
can you open an issue to not forget about this?

Reply:
https://github.com/camptocamp/docker-odoo-project/issues/108